pkg/provisioners/provisioner_util.go

```diff
@@ -69,13 +78,24 @@ func newDispatcherService(ccp *eventingv1alpha1.ClusterChannelProvisioner) *core
 			Selector: labels,
 			Ports: []corev1.ServicePort{
 				{
-					Name:       "http",
+					// There is a bug in Istio where named port doesn't work when connecting using an ExternalName service
+					// Refer to https://github.com/istio/istio/issues/13193 for more details.
+					// TODO: Uncomment Name:"http" when ISTIO fixes the issue
```

Comment:
Does this need to be copied to the comment I made above?

Reply:
Yes this is the reason I removed it. Didn't put the comment in UT, because when we revert this UTs will anyways fail and we will have to update UTs.